### PR TITLE
Add warning about using spaces in IPA filename.

### PR DESCRIPTION
**Pull request by njt1982**
See: https://github.com/godotengine/godot/issues/30052

### CODE REVIEW

**Comment by LikeLakers2** [comment]
While I'm not an iOS dev, I do question if this is something that could be fixed, rather than having to warn the user about the issue.

**Comment by njt1982** [comment]
@LikeLakers2 I agree about it being fixed in the engine being the ideal outcome, but in the meantime I think users could benefit from a heads up about the consequences of chosing spaces ;)

File: getting_started/workflow/export/exporting_for_ios.rst [hunk 1]
```diff
@@ -39,6 +39,10 @@ After you click **Export Project**, there are still two important options left:
           project's name. When you see **exported_xcode_project_name**
           in the following steps, replace it with the name you used instead.
 
+.. note:: Avoid using spaces when you chose your **exported_xcode_project_name** as
+          this can lead to corruption in your XCode project file.
```
Comment (cbscribe) [inline getting_started/workflow/export/exporting_for_ios.rst]: This should be "choose" not "chose".

Comment (akien-mga) [reply getting_started/workflow/export/exporting_for_ios.rst]: Fixed.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**